services/src/tests/e2e/jest-e2e-environment.js

```diff
@@ -1,16 +1,16 @@
 const NodeEnvironment = require('jest-environment-node');
 const dockerCompose = require('docker-compose');
-const waitFor = require('./waitFor');
+const waitFor = require('./wait-for');
 
-const sleep = timeout => new Promise(r => setTimeout(r, timeout));
+const sleep = (timeout) => new Promise((r) => setTimeout(r, timeout));
```

Comment:
Why do we require parentheses for a single param? it's kind of annoying and clutters the code

Reply:
It's prettier default. I didn't do anything about this. I'll see if it can be changed

Reply:
Ok. Found: https://prettier.io/docs/en/options.html#arrow-function-parentheses
Fixed